src/main/java/at/hannibal2/skyhanni/features/garden/composter/ComposterOverlay.kt

```diff
@@ -379,22 +394,32 @@ class ComposterOverlay {
         return price
     }
 
+    @SubscribeEvent
+    fun onRepoReload(event: io.github.moulberry.notenoughupdates.events.RepositoryReloadEvent) {
+        updateOrganicMatterFactors()
+    }
+
     @SubscribeEvent
     fun onRepoReload(event: RepositoryReloadEvent) {
-        try {
-            val garden = event.getConstant("Garden")!!
+        garden = event.getConstant("Garden")!!
+        updateOrganicMatterFactors()
+    }
 
-            val baseValues = mutableMapOf<String, Double>()
-            for ((name, value) in garden["organic_matter"].asJsonObject.entrySet()) {
-                baseValues[name] = value.asDouble
-            }
-            organicMatterFactors = updateOrganicMatterFactors(baseValues)
+    private fun updateOrganicMatterFactors() {
+        try {
+            garden?.let {
+                val baseValues = mutableMapOf<String, Double>()
+                for ((name, value) in it["organic_matter"].asJsonObject.entrySet()) {
+                    baseValues[name] = value.asDouble
+                }
+                organicMatterFactors = updateOrganicMatterFactors(baseValues)
```

Comment:
These can probably be inlined: 
```suggestion
                organicMatterFactors = updateOrganicMatterFactors(it["organic_matter"].asJsonObject.entrySet().associate { it.key to it.value.asDouble })
```

Reply:
I think i have implemented that correctly

Reply:
done